js1/funkce-obory/zastinovani.md

```diff
@@ -14,18 +14,18 @@ if (age < 18) {
 }
 ```
 
-Pravidlo při hledání proměnných říká, že se použije ta deklarace, na kterou runtime při procházení nadřazených bloků narazí nejdříve. Díky tomu, že se prohledává vždy od nejnižšího patra k nejvyššímu, v bloku `if` narazíme nejdřív na globální proměnnou :var[message]. Naopak v bloku `else` dříve najdeme lokální proměnnou. Tomuto principu se říká :term{cs="zastínění" en="shadowing"}. Proměnná, která je z hlediska hierarchie bloků níže, takzvaně zastíní stejně pojmenovou proměnnou, která se nachází výše.
+Pravidlo při hledání proměnných říká, že se použije ta deklarace, na kterou runtime při procházení nadřazených bloků narazí nejdříve. Díky tomu, že se prohledává vždy od nejnižšího patra k nejvyššímu, v bloku `if` narazíme nejdřív na globální proměnnou `message`. Naopak v bloku `else` dříve najdeme lokální proměnnou. Tomuto principu se říká :term{cs="zastínění" en="shadowing"}. Proměnná, která je z hlediska hierarchie bloků níže, takzvaně _zastíní_ stejně pojmenovou proměnnou, která se nachází výše.
 
 V praxi je nejlepší, když má náš program tak dobře pojmenované proměnné, že se nevzájem nazastiňují. Zjednodušujeme tak práci všem čtenářům, kteří tak mají o starost méně při louskání našeho kódu. Rozhodně je ale dobré vědět, že zastínění může nastat a JavaScript runtime se s ním snadno vypořádá.
 
 ### Obory platnosti a funkce
 
-Jak po předchozích lekcích už všichni víme, bloky kódu se používají také k vytváření funkci. Zde do oborů platnosti vstupuje další hráč, a to jsou parametry funkce. Ty se z hlediska hierarchie nacházejí jakoby na rozhraní mezi blokem funkce a jeho nadřazeným blokem. Prohlédněte si porozně následující kód.
+Jak po předchozích lekcích už všichni víme, bloky kódu se používají také k vytváření funkci. Zde do oborů platnosti vstupuje další hráč, a to jsou parametry funkce. Jsou napsané ještě před blokem funkce, před otevírací závorkou `{`. Z hlediska oboru platnosti se ale chovají tak, jako by byly uvnitř bloku funkce, na jeho samém začátku. Prohlédněte si porozně následující kód:
 
 ```js
 const message = 'Vítej ve světě slasti';
 
-const checkAge = (age, message) => {
+const checkAge = (age, message) => { // Z hlediska oboru platnosti jsou „age“ a „message“ jakoby definovány až zde, uvnitř bloku funkce 
```

Comment:
Nedával by ten komentář větší smysl až na dalším řádku? Možná je to jen můj osobní pocit, ale moc tam nevidím ten záměr, že je komentář až za závorkou.